packages/contracts-bedrock/src/L1/FeesDepositor.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { IL1CrossDomainMessenger } from "interfaces/L1/IL1CrossDomainMessenger.sol";
+import { ProxyAdminOwnedBase } from "src/L1/ProxyAdminOwnedBase.sol";
+import { ReinitializableBase } from "src/universal/ReinitializableBase.sol";
+import { ISemver } from "interfaces/universal/ISemver.sol";
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+/// @custom:proxied true
+/// @title FeesDepositor
+/// @notice A contract that deposits fees to the L2 recipient when the deposit threshold is reached.
+contract FeesDepositor is ProxyAdminOwnedBase, Initializable, ReinitializableBase, ISemver {
+    /// @notice The L1CrossDomainMessenger contract.
+    IL1CrossDomainMessenger public messenger;
+
+    /// @notice The threshold at which fees are deposited.
+    uint96 public minDepositAmount;
```

Comment:
Specs mention uint256 instead uint96, for the sake of consistency we should update the sepcs